### PR TITLE
Fast deceleration, slower acceleration more intuitive for new pilots

### PR DESCRIPTION
**Pull request by MaEtUgR**
added separate deceleration slewrate for velocity control

to improve the smooth user experience while accelerating
but not have any delay when braking

### CODE REVIEW

**Comment by MaEtUgR** [comment]
Here's a plot of the x axis velocity controller in action with a lot lower acceleration than deceleration.
deceleration_steeper

File: src/modules/mc_pos_control/mc_pos_control_main.cpp [hunk 1]
```diff
@@ -660,7 +662,7 @@ MulticopterPositionControl::parameters_update(bool force)
 		 * increase the maximum horizontal acceleration such that stopping
 		 * within 1 s from full speed is feasible
 		 */
-		_params.acc_hor_max = math::max(_params.vel_cruise(0), _params.acc_hor_max);
+		//_params.acc_hor_max = math::max(_params.vel_cruise(0), _params.acc_hor_max);
```
Comment (Stifael) [inline src/modules/mc_pos_control/mc_pos_control_main.cpp]: Is this still required? can we not just delete it?

Comment (MaEtUgR) [inline src/modules/mc_pos_control/mc_pos_control_main.cpp]: sure, anyways had to rebase

**Comment by MaEtUgR** [comment]
btw this pr solves https://github.com/PX4/Firmware/issues/6702 with reasonable parameters (like the defaults)

**Comment by MaEtUgR** [comment]
Closing in favour of https://github.com/PX4/Firmware/pull/6870

**Pull request closed without merging**